google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/config/tag_template_constants.py

```diff
@@ -24,6 +25,34 @@ def __init__(self, name, display_name, field_type, is_required=False):
             self.type = field_type
             self.is_required = is_required
 
+    class ClusterFields:
+
+        def __init__(self, num_brokers, num_topics, bootstrap_address):
+            self.num_brokers = num_brokers
+            self.num_topics = num_topics
+            self.bootstrap_address = bootstrap_address
+
+    class TopicFields:
+
+        def __init__(self, num_partitions, retention_time,
```

Comment:
should the fields be passed from the outside? Can we just create them in a (no-arg) constructor? Is there any case where we would want to pass different values based on some condition?

Reply:
not really, we can create them straight in the constructor. I think your approach actually creates more understandable outcome